driver/config/config.cpp

```diff
@@ -97,3 +99,270 @@ void writeDSNinfo(const ConnInfo * ci) {
 
 #undef WRITE_CONFIG
 }
+
+key_value_map_t readConnectionString(const std::string & connection_string) {
```

Comment:
I would really love to see some explanatory comment on functions like these, something like:
/**
Extract and validate keys and values from DSN string. Keys are parsed from string and assumed to be in the following format:
key=value;

Please note that keys case sensitive, and  keys with no values are expected to be in following format...
"

Reply:
I'll add that.